mattumland/next-graphcms-shows
# Add function to conditionally render truthy links

## What issue does this solve?
https://codevalapp.github.io/next-graphcms-shows/issues/5/

## Background
Not every artist has all four links (website, facebook, instagram, youTube). We only want to render links that are live. This adds a function to `[slug].js` that checks if a link is truthy and returns a link if is does. 

## Manual testing instructions
1. Boot up dev server
2. Go to `shows`
3. Click on multiple shows, confirm that only live links are rendered under the artist bio.
